src/main/scala/no/ndla/learningpathapi/model/domain/EmbedUrl.scala

```diff
@@ -10,6 +10,26 @@ package no.ndla.learningpathapi.model.domain
 
 import no.ndla.learningpathapi.model.api.ValidationMessage
 
-case class EmbedUrl(url:String, language:Option[String], embedType: String) extends LanguageField {
+case class EmbedUrl(url:String, language:Option[String], embedType: EmbedType.Value) extends LanguageField {
   override def value = url
-}
\ No newline at end of file
+}
+
+object EmbedType extends Enumeration {
+
+  val oembed, lti = Value
```

Comment:
I følge Scala sin style-guide så bør også konstanter starte med stor forbokstav og deretter camel-casing.
(http://docs.scala-lang.org/style/naming-conventions.html)

I dette tilfellet synes jeg at vi bør bruke OEmbed og siden LTI er en forkortelse, så kan LTI være i store bokstaver.

Om vi vil at verdien ut i APIet og i databasen skal være med små bokstaver, så kan dette sørges for slik:

```
val OEmbed = Value("oembed")
val LTI = Value("lti")
```